components/scheduler/views/month.md

```diff
@@ -14,19 +14,27 @@ The Month view of the Scheduler for Blazor shows an entire month to the user.
 
 The `Date` parameter of the Scheduler controls which month is displayed. It's the one containing the date.
 
-Each day slot shows up to two appointments. If there are more appointments for a day, an ellipsis button will provide access to the DayView for the specific day. This avoids cluttering. You can also access the day view from the day number. Of course, you must also [define a day view]({%slug scheduler-views-day%}) so the user can see it.
-
-Here is how the Scheduler determines which two appointments to display in a month view slot:
-
-1. The Scheduler sorts all appointments in the current month by start date (ascending) and then by end date (descending).
-1. It takes the first two appointments for each day from the sorted collection.
-1. The component sorts the two appointments for each day by start time (ascending) and then by end time (descending).
 
 In this article:
 
+* [View Parameters](#view-parameters)
 * [Example](#example)
 * [Resource Grouping](#resource-grouping-in-the-month-view)
 
+## View Parameters
+
+The following parameters allow you to configure the month view:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Attribute | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `ItemsPerSlot` | `int` <br /> (`2`) | Indicates the number of appointments that can be displayed per day. |
+
+
+If there are more appointments for a day than per the `ItemsPerSlot` parameter, an ellipsis button will provide access to the DayView for the specific day. You must [define a day view]({%slug scheduler-views-day%}) so the user can see it. The Scheduler sorts and displays the number of appointments per the `ItemsPerSlot` parameter, by start time (ascending) and then by end time (descending).
```

Comment:
Present Simple is preferable to other tenses. You can safely switch to Present Simple throughout lines 35 and 37.

See [here ](https://docs.telerik.com/style-guide/basic-rules-and-guidelines#use-simple-tenses) for reference.